src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/EmailHistoryService.java

```diff
@@ -0,0 +1,48 @@
+package ca.bc.gov.open.jag.tco.oracledataapi.service;
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import ca.bc.gov.open.jag.tco.oracledataapi.model.EmailHistory;
+import ca.bc.gov.open.jag.tco.oracledataapi.repository.EmailHistoryRepository;
+
+@Service
+public class EmailHistoryService {
+
+	@Autowired
+	EmailHistoryRepository emailHistoryRepository;
+	
+	@PersistenceContext
+    private EntityManager entityManager;
```

Comment:
I don't think we need the Entity Manager in this service since the emailHistoryRepository suffice for inserting the EmailHistory and returning the id of it so it can be removed.